osd/entrypoint.sh

```diff
@@ -18,6 +18,7 @@ do
 
    # Check to see if our OSD has been initialized
    if [ ! -e /var/lib/ceph/osd/${CLUSTER}-${OSD_ID}/keyring ]; then
+      ceph osd create
```

Comment:
Not sure about this one, if we have to use it, this should be after the client.admin.keyring check.
I also think you should catch the OSD_ID at some point.
@Ulexus what do you think? Did we have any good reason to not use the command in the entrypoint?